src/WebUI/Common/KubeSummary.tsx

```diff
@@ -67,6 +72,8 @@ export interface IKubeSummaryProps extends IVssComponentProperties {
 export class KubeSummary extends BaseComponent<IKubeSummaryProps, IKubernetesContainerState> {
     constructor(props: IKubeSummaryProps) {
         super(props, {});
+        KubeSummary._imageService = this.props.imageService;
```

Comment:
we should call, something like, update or like that, so we do not access the prive variables. make this as a private static

Reply:
imageService & kubeservice are private static